docs/docs/README.md

```diff
@@ -39,6 +39,15 @@ The diagram below provides a high level overview of what a typical deployment wi
 
     ![architecture](assets/images/jira/Terraform.png "Deployment")
 
+!!! tip "Cluster size and cost"
+
+    In the installation process, [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
```

Comment:
Not sure about this sentence, maybe:

[Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) is provisioned as part of the deployment